### PR TITLE
feat: implement angle inputs

### PR DESCRIPTION
**Pull request by jrandolf**

### CODE REVIEW

File: src/bidiMapper/domains/input/ActionDispatcher.ts [hunk 1]
```diff
@@ -822,3 +820,70 @@ function getCdpButton(button: number) {
...
+    let tiltXRadians = 0;
+    let tiltYRadians = 0;
+
+    if (altitudeAngle === 0) {
+      // the pen is in the X-Y plane
+      if (azimuthAngle === 0 || azimuthAngle === 2 * Math.PI) {
+        // pen is on positive X axis
+        tiltXRadians = Math.PI / 2;
+      }
+      if (azimuthAngle === Math.PI / 2) {
```
Comment (OrKoN) [inline src/bidiMapper/domains/input/ActionDispatcher.ts:848]: does this comparison on floats work? I think we need to compare with some precision

**Review by OrKoN — CHANGES_REQUESTED** [review]
is it tested by WPT? let's enable them here or add e2e tests.

File: src/bidiMapper/domains/input/ActionDispatcher.ts (hunk 1 above)
Comment (jrandolf) [reply src/bidiMapper/domains/input/ActionDispatcher.ts:848]:
```
6.283185307179586 === Math.PI * 2 // true
```

Comment (OrKoN) [reply src/bidiMapper/domains/input/ActionDispatcher.ts:848]: what about `6.2831853071795867` or `6.2831853071795864`?

Comment (OrKoN) [reply src/bidiMapper/domains/input/ActionDispatcher.ts:848]: Interesting though, I guess this works as expected 👍

**Review by OrKoN — APPROVED** [review]

**Pull request merged**